docs/source/hpxml_to_openstudio.rst

```diff
@@ -554,13 +553,26 @@ CFA = conditioned floor area [ft²],
 NCfl = number of conditioned floor levels number of conditioned floor levels in the residence, including conditioned basements, 
 bsmnt = presence = 1.0 or absence = 0.0 of an unconditioned basement in the residence.
 
-For a ``SystemType/Recirculation`` system, the following elements are required:
+For a ``SystemType/Recirculation`` system, the following element is required:
```

Comment:
Rather than splitting the list into required and optional inputs for recirculation systems, how about using a single list where you call out certain inputs as optional? See this [simple solar hot water](https://github.com/NREL/OpenStudio-HPXML/blob/solar_fraction/docs/source/hpxml_to_openstudio.rst#solar-thermal) section where ConnectedTo is called out as optional.